<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1288</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{	
	font: 700 10pt &quot;Consolas&quot;;
	border-radius:35px;
}
QPushButton{
	color : rgb(40, 40, 40);
	background-color: lightgray;
	border: 1px solid lightgray; 
	padding: 1px;
	z-index: 200;
}
QPushButton:pressed {
	color: black;
	background-color: lightyellow;
	border: 1px solid lightyellow; 
	border-radius: 4px;
	z-index: 200;
}
QLineEdit{
	background-color: #ffffff;
	border-radius: 5px;
}
QLabel{
	color:  #ddd;
	background-color: rgb(116, 116, 116);
	border-radius: 4px;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="InfoTextEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>501</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>431</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QTextEdit" name="RxTextEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="cleanRxPushButton">
       <property name="text">
        <string>CLEAN Rx</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="OpenPortcomboBox">
             <item>
              <property name="text">
               <string>COM1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>COM2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>COM3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>COM4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>COM5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>COM6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>COM7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>COM8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>COM9</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="CMDcomboBox">
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <property name="placeholderText">
              <string>CMD</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QPushButton" name="SendPushButton">
           <property name="text">
            <string>SEND CMD</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="openPortPushButton">
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color-background: gray;
}</string>
           </property>
           <property name="text">
            <string>OPEN</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="GetAlivePushButton">
         <property name="text">
          <string>ALIVE</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="TxTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cleanTxPushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>540</y>
      <width>60</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CLEAN Tx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>     MANUAL COMMANDS</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>247</width>
      <height>115</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>     SERVO</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="SETSERVOLABEL">
           <property name="text">
            <string> SET SERVO ANGLE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="SetServoDoubleSpinBox">
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="minimum">
            <double>-180.000000000000000</double>
           </property>
           <property name="maximum">
            <double>180.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SetServoAnglePushButton">
           <property name="text">
            <string>SET SERVO ANGLE</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="SERVOANGLE_label">
             <property name="text">
              <string> CURRENT ANGLE</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="SetServoAngleLineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string> Max &amp; Min Angles</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="RightServoPushButton">
               <property name="text">
                <string>&lt;-</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="LeftServoPushButton">
               <property name="text">
                <string>-&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>241</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>   ULTRASONIC</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="LASTDISTANCE_label">
           <property name="text">
            <string>  LAST DISTANCE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LastDistance_LineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="GetDistancePushButton">
         <property name="text">
          <string>GET DISTANCE</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>259</width>
      <height>115</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>   IR SENSORS</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <widget class="QLineEdit" name="IR_1_LineEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="IR_1_label">
         <property name="text">
          <string>  IR_1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="IR_2_label">
         <property name="text">
          <string>  IR_2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="IR_2_LineEdit"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="IR_3_label">
         <property name="text">
          <string>  IR_3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="IR_3_LineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="1" column="1">
        <widget class="QComboBox" name="IR_ComboBox">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>SENSOR IR SELECTED TO SET </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="SetBlackPushButton">
         <property name="text">
          <string>SET BLACK</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="SetWhitePushButton">
         <property name="text">
          <string>SET WHITE</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>     MOTORS</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QLabel" name="M1_label">
         <property name="text">
          <string>  M1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="M1_LineEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="M2_LineEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="M1_label_2">
         <property name="text">
          <string>  M1</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QSlider" name="verticalSlider">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="verticalSlider_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>510</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QPushButton" name="ModePushButton">
       <property name="text">
        <string>CHANGE MODE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ModeLineEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1288</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
